dotCMS/src/main/java/com/dotmarketing/common/reindex/ReindexThread.java

```diff
@@ -669,37 +672,63 @@ private void fillRemoteQ (boolean includeFailedRecords) throws DotDataException
 	    }
 	}
 
+	@CloseDBIfOpened
+	private List<Map<String,String>> getContentletVersionInfoByIdentifier (final String id) throws DotDataException {
+
+		final String sql = "select working_inode,live_inode from contentlet_version_info where identifier=?";
+		final DotConnect dc = new DotConnect();
+		dc.setSQL(sql);
+		dc.addParam(id);
+		return dc.loadResults();
+	}
+
+	@CloseDBIfOpened
+	private com.dotmarketing.portlets.contentlet.business.Contentlet getContentletByINode (final String inode) throws DotHibernateException {
+
+		return (com.dotmarketing.portlets.contentlet.business.Contentlet)
+				HibernateUtil.load(com.dotmarketing.portlets.contentlet.business.Contentlet.class, inode);
+	}
+
+	@CloseDBIfOpened
+	private Contentlet convertFatContentletToContentlet (final com.dotmarketing.portlets.contentlet.business.Contentlet fattyContentlet) throws DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 181 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)